table/table_builder.h

```diff
@@ -113,7 +113,9 @@ struct TableBuilderOptions {
       const int64_t _oldest_key_time = 0,
       const uint64_t _file_creation_time = 0, const std::string& _db_id = "",
       const std::string& _db_session_id = "",
-      const uint64_t _target_file_size = 0, const uint64_t _cur_file_num = 0)
+      const uint64_t _target_file_size = 0, const uint64_t _cur_file_num = 0,
+      const SequenceNumber _last_level_inclusive_max_seqno_threshold =
```

Comment:
I understand it's needed for the properties collector (context) but it just seems weird to be in TableBuilderOptions. Is there a way this information could come in through the collector factory or collector? Hmm, probably not, but maybe it would make sense for TableBuilderOptions to extend or include TablePropertiesCollectorFactory::Context, so that it's clear why this field is needed?

Or maybe the conversion from collector factories to collectors could be done outside the table builders and just pass in the set of collectors.

Not a big deal. In either case, now or later, I don't see a good route to *not* requiring this info in TablePropertiesCollectorFactory::Context.